share/ruby-install/package_manager.sh

```diff
@@ -41,7 +41,10 @@ function install_packages()
 			run $brew_sudo brew upgrade "$@" || return $?
 			;;
 		pacman)
-			local missing_pkgs=($(pacman -T "$@"))
+			local missing_pkgs=()
+			while IFS='' read -r line; do
+				missing_pkgs+=("$line")
```

Comment:
I'm not a Arch user so I can't verify this, can you double check if `pacman -T` outputs the missing packages one per-line or a single-line with the package names separated by spaces?
https://archlinux.org/pacman/pacman.8.html

Reply:
Don't have Arch installed, either. I'm just maintaining Gentoo ebuild in [::guru overlay](https://github.com/gentoo/guru/tree/dev/dev-ruby/ruby-install), and currently I simply disable `make check` :D

Reply:
Wait, I forgot I can check using the [archlinux docker image](https://hub.docker.com/_/archlinux).